arch/build.sh

```diff
@@ -77,15 +77,15 @@ if [[ "${future}" == "yes" ]]; then
     if [[ ${specs_path} != ${tmp_specs_path} ]]; then
         cp -r ${specs_path} "${tmp_specs_path}"
     fi
```

Comment:
Is this conditional necessary? Line 80 cp command is the same as the command inside the conditional

Reply:
Done [c28ed20](https://github.com/wazuh/wazuh-packages/pull/1579/commits/c28ed207f9bcada77bfb5ec27de04e67ea97c9db)